submission/persistence/src/test/kotlin/ac/uk/ebi/biostd/persistence/mapping/extended/to/ToExtFilesTest.kt

```diff
@@ -0,0 +1,50 @@
+package ac.uk.ebi.biostd.persistence.mapping.extended.to
+
+import ac.uk.ebi.biostd.persistence.common.NO_TABLE_INDEX
+import ac.uk.ebi.biostd.persistence.mapping.extended.to.test.assertExtFile
+import ac.uk.ebi.biostd.persistence.mapping.extended.to.test.fileDb
+import ebi.ac.uk.asserts.assertThat
+import ebi.ac.uk.io.sources.FilesSource
+import ebi.ac.uk.util.collections.second
+import io.github.glytching.junit.extension.folder.TemporaryFolder
+import io.github.glytching.junit.extension.folder.TemporaryFolderExtension
+import io.mockk.every
+import io.mockk.junit5.MockKExtension
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.assertj.core.api.Assertions.assertThat as assertThat1
+
+private const val FILE_1 = "file1"
+private const val FILE_2 = "file2"
+private const val FILE_3 = "file3"
+
+@ExtendWith(value = [MockKExtension::class, TemporaryFolderExtension::class])
```

Comment:
1) You don't need to create an array, just use:
`@ExtendWith(MockKExtension::class, TemporaryFolderExtension::class)`

2) You are not using `MockKExtension` in this test